reana_client/cli/workflow.py

```diff
@@ -349,8 +362,13 @@ def workflow_start(ctx, workflow, access_token,
     help='Set status information verbosity.')
 @click.pass_context
 def workflow_status(ctx, workflow, _filter, output_format,
-                    access_token, verbose):
-    """Get status of previously created workflow."""
+                    access_token, verbose):  # noqa: D301
+    """Get status of previously created workflow.
+
```

Comment:
Add some body, e.g.:

Retrieve status of a previously created workflow.  The status can be created, queued, running, failed, etc.  You can increase verbosity or filter retrieved information by passing appropriate command-line options.